index.js

```diff
@@ -169,4 +169,35 @@ const isEmptyMapOrSet = x => (is.map(x) || is.set(x)) && x.size === 0;
 
 is.empty = x => !x || isEmptyStringOrArray(x) || isEmptyObject(x) || isEmptyMapOrSet(x);
 
+const isAnyOrAll = (all, predicate, values) => {
+	// `values` is the calling function's "arguments object".
+	// We have to do it this way to keep node v4 support.
+	// So here we convert it to an array and slice off the first item.
+	values = Array.prototype.slice.call(values, 1);
+
+	if (is.function(predicate) === false) {
+		throw new TypeError(`Invalid predicate: ${util.inspect(predicate)}`);
+	}
+
+	if (values.length === 0) {
+		throw new TypeError(`Invalid number of values: ${util.inspect(values.length)}`);
```

Comment:
You can never really have a value other than `0`, so why call inspect on this? instead, make the error more readable, f.e: `No values passed to ${callerMethod}`, where `callerMethod` can be passed as an argument.